### PR TITLE
Return a WordResult object

### LINKED ISSUE #454: Bug while converting emoji shortnames
## Expected Behavior
`:smile:` -> :smile: 

## Current Behavior
If you use the blank key after you wrote `:smile:` very quickly, you get ::smile: as result (notice the leading colon). If you wait a few deciseconds after you wrote `:smile:` everything is okay.

Can you reproduce?

* Threema Web version: 1.8.2
* Browser name and version: Firefox 58.0.2

### PR DESCRIPTION
**Pull request by sillych**
fix #454

### CODE REVIEW

**Comment by dbrgn** [comment]
> fox #454

:fox_face:

**Comment by sillych** [comment]
:sweat_smile:

File: src/threema.d.ts [hunk 1]
```diff
@@ -495,6 +495,11 @@ declare namespace threema {
         isAll: boolean;
     }
 
+    interface WordResult {
+        word: string;
+        realLength: number;
```
Comment (dbrgn) [inline src/threema.d.ts]: What exactly does `realLength` mean? From the tests it looks like it also includes trailing whitespace?

Comment (sillych) [reply src/threema.d.ts]: Length of the untrimmed word

Comment (dbrgn) [reply src/threema.d.ts]: I pushed a commit with some code comments: https://github.com/threema-ch/threema-web/pull/456/commits/22dde4b6c396acefd5c4f662aef2f8461184a321 Are they correct?

Comment (sillych) [reply src/threema.d.ts]: :1st_place_medal:

**Pull request merged**